src/js/displayresults.js

```diff
@@ -0,0 +1,71 @@
+import dateBuilder from './datebuilder';
+import converter from './converter';
+
+const displayResults = weather => {
+  const tempVal = Math.round(weather.main.temp);
+  const minVal = Math.round(weather.main.temp_min);
+  const maxVal = Math.round(weather.main.temp_max);
+  const speed = Math.round(Math.round(weather.wind.speed));
+  const city = `${weather.name}, ${weather.sys.country}`;
+  const date = new Date();
+  const currentDate = dateBuilder(date);
+  const weatherType = weather.weather[0].main;
+  const { icon } = weather.weather[0];
+  const humidity = Math.round(weather.main.humidity);
+  const notification = document.querySelector('#notification');
+
+  notification.classList.add('d_none');
+
+  const tempValFar = converter(tempVal).farenheit;
+  const minValFar = converter(minVal).farenheit;
+  const maxValFar = converter(maxVal).farenheit;
+  const speedMph = converter(speed).mph;
+  const main = document.querySelector('#main');
```

Comment:
Can we refactor this logic into a function and move it out of here? 

Let's create a function that will clean the data, process it to what you want to show to your user and return a structured data for it to be populated and displayed to the DOM.

Let's add test for the refactored function.

Reply:
I'll improve on this